<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiikennepaikkaDialogi</class>
 <widget class="QDialog" name="LiikennepaikkaDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Lyhenne</string>
         </property>
         <property name="buddy">
          <cstring>lyhenneEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lyhenneEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nimi</string>
         </property>
         <property name="buddy">
          <cstring>nimiEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nimiEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ratakilometrit</string>
         </property>
         <property name="buddy">
          <cstring>kmSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="kmSpin">
         <property name="suffix">
          <string> km</string>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1299.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sijainti</string>
         </property>
         <property name="buddy">
          <cstring>sijaintiEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="sijaintiEdit">
         <property name="inputMask">
          <string>P 00:00:00 °,  I 00:00:00 ° ; </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="liikenteenohjausCheck">
         <property name="text">
          <string>Liikenteenohjaus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="taajamajunaCheck">
         <property name="text">
          <string>Taajamajunia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pikajunaCheck">
         <property name="text">
          <string>Pikajunia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pendolinoCheck">
         <property name="text">
          <string>Pendolinoja</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="peruNappi">
       <property name="text">
        <string>Peru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tallennaButton">
       <property name="text">
        <string>Tallenna</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tallennaButton</sender>
   <signal>clicked()</signal>
   <receiver>LiikennepaikkaDialogi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
